cli/src/tests/corpus_test.rs

```diff
@@ -390,7 +390,9 @@ fn flatten_tests(test: TestEntry) -> Vec<(String, Vec<u8>, String, bool)> {
                 }
                 result.push((name, input, output, has_fields));
             }
-            TestEntry::Group { mut name, children, .. } => {
+            TestEntry::Group {
+                mut name, children, ..
+            } => {
```

Comment:
For some of these files, it looks like the only changes are formatting. Could you remove these changes, so that the only files affected by the PR are the ones with code changes?